deployment/README.md

```diff
@@ -6,6 +6,26 @@ export GOOGLE_CLOUD_PROJECT=<PROJECT_ID>
 ```
 The official project id of OSV is `oss-vdb`.
 
+## API Domain Name
+
+Due to [terraform complexities](https://github.com/hashicorp/terraform-provider-google/issues/5528),
+setting up the OSV API requires a custom domain to serve it on.
+
+1. Verify the ownership of your domain:
+  
+    Go to 
+
+    `https://www.google.com/webmasters/verification/verification?authuser=0&domain=custom.domain.name`
+
+    replacing `custom.domain.name` with the domain to be verified.
+    
+    (This link is usually generated when adding a domain mapping to a service in Cloud Run.
+    I don't know how to navigate to that page otherwise. Trying to add a property from
+    [Webmaster Central](https://www.google.com/webmasters/verification/home)
+    adds it as a site, rather than as a domain.)
+
+2. Add DNS CNAME record mapping the API URL to `ghs.googlehosted.com.`
```

Comment:
Is this a different URL from `custom.domain.name` ? If so, let's make this clearer.